types/stylelint/index.d.ts

```diff
@@ -844,7 +844,7 @@ declare namespace stylelint {
 		| {
 				word: string;
 		  }
-		| {};
+		| object;
```

Comment:
Like I said in https://github.com/stylelint/stylelint/issues/8126#issuecomment-2559331560 you probably want `{ [key: string]: never }` or `Record<string, never>`.

ref https://biomejs.dev/linter/rules/no-banned-types/

Reply:
Neither actually works :)
`object` does work.

<img width="817" alt="Screenshot 2024-12-24 at 14 27 26" src="https://github.com/user-attachments/assets/a3bd04a6-152f-4529-8ea1-5f1e6118be3b" />